lib/unsanitized.js

```diff
@@ -0,0 +1,83 @@
+// names of escaping functions that we acknowledge
+const VALID_ESCAPERS = ["Sanitizer.escapeHTML", "escapeHTML"];
+const VALID_UNWRAPPERS = ["Sanitizer.unwrapSafeHTML", "unwrapSafeHTML"];
+
+class Unsanitized {
```

Comment:
Agreed on the class name... throw me some suggestions.

If we choose not to pass in context here the boiler plate in the rules to handle the return value would be huge, so I think this abstracted code always needs context.

An alternative is perhaps making both functions passed a object of params instead of the list of args.
For example (ES6 makes this less ugly too):
```
reportUnsupported({
  node,
  reason,
  errorTitle,
  context
})
```

Another alternative in the rules is to do this:

```
const ruleInstance = new RuleHandler(context);
return {
  AssignmentExpression: ruleInstance.AssignmentExpression
}
```

But somewhere probably in this include we are likely to want to parse the supplied config and run regexs etc.